src/components/TaskGroupProgress/index.jsx

```diff
@@ -244,13 +245,29 @@ export default class TaskGroupProgress extends Component {
     }
   };
 
+  getFaviconStatus = statusCount => {
```

Comment:
You can access `statusCount` via `const { statusCount } = this.state;` instead of making it an argument to the method.